shared/panels/issue-creation-panel/_features/markdown-editor/markdown-editor.tsx

```
@@ -0,0 +1,36 @@
+            commands.divider,
+            commands.table,
+            commands.codeBlock,
+            commands.code,
+            commands.quote,
+          ]}
+          extraCommands={[commands.fullscreen]}
+        />
+      </div>
+    );
+  }
+);
```

ESLint

[error] Component definition is missing display name

(react/display-name)